docs/testplan/Smartswitch-test-plan.md

```diff
@@ -56,12 +74,21 @@ General convention of DPU0, DPU1, DPU2 and DPUX has been followed to represent D
 | 1.4 | Check DPU console       | To Verify console access for all DPUs       | |
 | 1.5 | Check midplane ip address between NPU and DPU      | To Verify PCIe interface created between NPU and DPU according to bus number | |
 | 1.6 | Check DPU shutdown and power up individually      |  To Verify DPU shutdown and DPUs power up | |
-| 1.7 | Check removal of pcie link between NPU and DPU       | To Verify the PCie hot plug functinality        | |
+| 1.7 | Check pcie link status between NPU and DPU       | To Verify the PCie hot plug functinality        | |
 | 1.8 | Check the NTP date and timezone between DPU and NPU       | To Verify NPU and DPU are in sync with respect to timezone and logs timestamp | |
 | 1.9 | Check the State of DPUs      | To Verify DPU state details during online and offline      | |
 | 1.10 | Check the Health of DPUs       | To Verify overall health (LED, process, docker, services and hw) of DPU | Phase:2 |
 | 1.11 | Check reboot cause history       | To Verify reboot cause history cli | |
 | 1.12 | Check the DPU state after OS reboot       | To Verify DPU state on host reboot | |
```

Comment:
Is this Switch reboot or DPU OS reboot, could you update the sentence accordingly.

Reply:
Changed it to say switch reboot